dandi/pynwb_utils.py

```diff
@@ -0,0 +1,16 @@
+from pynwb import NWBHDF5IO
+
+
+def get_metadata(filepath):
+    out = dict()
+    with NWBHDF5IO(filepath, 'r') as io:
+        nwb = io.read()
+        out['publications'] = nwb.related_publications
+        out['lab'] = nwb.lab
+        out['experimenters'] = nwb.experimenter
+        out['keywords'] = nwb.keywords
+        out['date'] = nwb.session_start_time
+        out['subject_id'] = nwb.subject.subject_id
```

Comment:
Shouldn't we get some session id as well?

Reply:
Yeah that could be useful. It's just as easy as the others